src/modules/ROOT/pages/changelog/202303/02.gen.adoc

```diff
@@ -0,0 +1,73 @@
+* Added option for transparent color in the color picker used in {APP_NAME_ADMIN}, and {APP_NAME_REPORTER}.
+The change was added to give users more configuration options when selecting color or background color for an element
+({CORTEZA_PULL_BASE}1211[`1211`]).
+* Added the option to set a record list filter as a preset filter for a user.
+The filter can be applied using the 'Filters' dropdown menu.
+The change was added to give the user the ability to pick a pre-configured filter
+({CORTEZA_PULL_BASE}1227[`1227`]).
+* Added menu with 'more' options (delete, un-delete, and permissions) to resource lists in {APP_NAME_COMPOSE}, {APP_NAME_ADMIN}, {APP_NAME_REPORTER}, and {APP_NAME_WORKFLOW}.
+The change was added so the user doesn't have to scroll horizontally to access all resource options
+({CORTEZA_COMMIT_BASE}46f5455[`46f5455`]).
+
+.[#2023_03_02-changed]#<<2023_03_02-changed,Changed:>>#
+* Changed the way reminders in {APP_NAME_COMPOSE} look and behave which provides an easier way for the user to work with.
+The change was made to be consistent with the rest of the {PRODUCT_NAME} UI and improve the UX
+({CORTEZA_PULL_BASE}1239[`1239`]).
+* Changed the feedback email domain in the page header of every {PRODUCT_NAME} web application from `crusttech` to `cortezaprojectorg`
+({CORTEZA_COMMIT_BASE}0072a4b[`0072a4b`]).
+* Changed reminders to clear out the snooze count after editing and modifying when the reminder should prompt
+({CORTEZA_COMMIT_BASE}256dd59[`256dd59`]).
+* Changed the format of displaying the total number of record list items - if N is number of records, then the new format would be N items to N records
+({CORTEZA_COMMIT_BASE}0fcbec0[`0fcbec0`]).
+* Changed the block magnify query param that keeps track of which block is magnified (zoomed in).
+The change was made to ensure that any future code implementations don't collide with the block magnify query param
+({CORTEZA_COMMIT_BASE}5fde4c9[`5fde4c9`]).
+* Changed the value of the `SMTP_FROM` variable in the server `env` variables
+The variable wasn't taken into consideration if a value was added to it in the server `env` file
+({CORTEZA_COMMIT_BASE}d2b389b[`d2b389b`]).
+
+.[#2023_03_02-fixed]#<<2023_03_02-fixed,Fixed:>>#
+* Fixed multiline privacy request comments displaying as inline comments.
+The fix was made by adding support for multiline comments in privacy requests
+({CORTEZA_PULL_BASE}1214[`1214`]).
+* Fixed {APP_NAME_COMPOSE} broken public icon url for page, namespace, and custom app icons after changing the Auth JWT secret.
+The fix was made by adding an exception for the namespace icons, page, and icon attachments
+({CORTEZA_PULL_BASE}1219[`1219`]).
+* Fixed available and selected fields not scrollable in safari due to set max height in the item picker component
+This issue was present in {APP_NAME_COMPOSE} and {APP_NAME_PRIVACY}
+({CORTEZA_PULL_BASE}1221[`1221`]).
+* Fixed funnel chart labels are hidden after hovering on them
```

Comment:
This one doesn't sound right; "Fixed funnel chart labels **are** hidden" -- the are part throws it off for me